pages/glossary/perceivable-content.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Perceivable content
+key: perceivable-content
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+A node is _perceivable content_ if all the following are true:
+
+- the node is [palpable content][]; and
+- the node is either [visible][] or [included in the accessibility tree][]; and
+- if the node is an element, it has a [semantic role][] which is neither `none` nor `presentation`.
+
+**Note:** Perceivable content corresponds to nodes that contain information and are perceived by some categories of users.
+
+#### Assumptions
+
+This definition assumes that elements with a [semantic role][] of `none` or `presentation` are [pure decoration][] and that elements which are [pure decoration][] either are not [included in the accessibility tree][] or have a [semantic role][] of `none` or `presentation`. Note that if this is not the case, then [Success Criterion 1.3.1: Info and Relationship][sc131] is likely not satisfied.
```

Comment:
This is weird (a = b then b = a or b = c)
And I don't understand the need for this assumption.